.travis.yml

```diff
@@ -37,7 +37,8 @@ branches:
         - /^\d.\d+$/
 
 before_script:
-    - travis_retry composer selfupdate
+    # To try to avoid running out of memory on PHP 5, and speed up install on all, using Composer 2.x here
+    - travis_retry composer self-update --preview
```

Comment:
_NOTE: This will be skipped on merge up to 2.0 as @alongosz was saying there where issues with Composer 2.0 on v3, which for some reason are not on v2.5._

Reply:
> _NOTE: This will be skipped on merge up to 2.0 as @alongosz was saying there where issues with Composer 2.0 on v3, which for some reason are not on v2.5._

If for this package there are no issues (tests are passing), then you might keep it.
We'll need to remove it anyway once composer 2 is released, because Travis already performs self-update, but stable only (AFAICS no way to configure it :( )